tarsier-service/service/src/db.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Any
+from sqlalchemy import create_engine, text, Row
+import uuid
+import json
+
+host = os.environ.get("DB_HOST") or "localhost"
+engine = create_engine(f"mysql+pymysql://user:password@{host}:3306/db") #docker compose ...@db:3306/db"
+
+with engine.connect() as conn:
+    # conn.execute(text('DROP TABLE jobs'))
+    conn.execute(text('''
+        CREATE TABLE IF NOT EXISTS jobs (
+            id VARCHAR(255) PRIMARY KEY,
```

Comment:
nit such a large column for an pk and lead to performance issues. If you want to use UUID as a pk binary(16) is probably better. eg:

```
create table tbl_test (
    guid binary(16) default (uuid_to_bin(uuid())) not null primary key,
    name varchar(50) not null
);
```

Ideally you would also want the db to control PK generation as done in the example above. For performance reason, is also quite nice to choose a pk that is ordered by time.

I'd recommend reading the below if curious :) 
https://planetscale.com/blog/why-we-chose-nanoids-for-planetscales-api
https://planetscale.com/blog/the-problem-with-using-a-uuid-primary-key-in-mysql